curriculum/challenges/english/07-scientific-computing-with-python/learn-tree-traversal-by-building-a-binary-search-tree/65c644829cfb63acf3479d09.md

```diff
@@ -1,63 +1,86 @@
 ---
-id: 655a452d40556e5c25e4aac8
-title: Step 26
+id: 65c644829cfb63acf3479d09
+title: Step 27
 challengeType: 20
-dashedName: step-26
+dashedName: step-27
 ---
 
 # --description--
 
-The next step is to work on the deletion of nodes.
+A list of `nodes` is given for you to insert into the `BinarySearchTree` instance `bst`.
 
-Within the `BinarySearchTree` class, define a `delete` method. It takes two parameters: `self` and `key`.
+You can insert multiple nodes into the tree like this:
 
-`key` is the value that the user wants to delete from the binary search tree.
+```python
+samples = [1, 2, 3, 4, 5]
+
+for sample in samples:
```

Comment:
It should be either `for node`  or `your_object.insert(sample)` to work. Anyway, I don't think the example is necessary. You can ask the camper to write a for loop to iterate over the list and call `self.insert` for each item.